### ISSUE TITLE
can not connect to QGC

### ISSUE THREAD

**Issue by feilongfl** [PX4/Flow#95]
i use the command below to upload firmware.and then i can not connect to the QGC,what should i do.
```
make px4flow-v1_default -j4 -l4 upload-usb
```
output:
![2017-08-18 19-19-17](https://user-images.githubusercontent.com/10269596/29456904-5aed2864-844a-11e7-9e4b-6c53bbe9f94a.png)

QGC:
![2017-08-18 19-17-32](https://user-images.githubusercontent.com/10269596/29456930-6c64f798-844a-11e7-9800-657d7c2b9cb2.png)

code is in master branch.

**Comment by feilongfl** [comment]
i have join the user group dialout

**Comment by feilongfl** [comment]
then i try to connect with mavproxy,it show me `Device or resource busy`
![2017-08-18 19-24-48](https://user-images.githubusercontent.com/10269596/29457085-0742c2a4-844b-11e7-952f-715b96a09718.png)

**Comment by ChristophTobler** [comment]
There is an error: `ERROR: timeout waiting for data`
 
It should be
```
Found board 6,0 bootloader rev 3 on /dev/ttyACM0
erase...
program...
verify...
done, rebooting.
```
Try again with `make upload-usb`

**Comment by feilongfl** [comment]
oh sorry, i put the wrong picture on it,it not the program downloads error.

i think it may be the QGC 's error.
i have try to use old QGC,it works,now i use the QGC version v3.1.1

**Comment by feilongfl** [comment]
i put an issues in [https://github.com/mavlink/qgroundcontrol/issues/5591](issues),they reply me it will be fix in v3.3
